akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -883,6 +1015,22 @@ class ClusterSingletonManager(
       if (m.uniqueAddress == cluster.selfUniqueAddress)
         logInfo("Self downed, waiting for removal")
       stay
+    case Event(ReleaseLeaseResult(result), _) ⇒
+      // TODO we could retry if false
+      logInfo("Lease release result [{}] in state [{}]", result, stateName)
```

Comment:
might be good to log error (similar as below) if result is false

Reply:
+1 to error